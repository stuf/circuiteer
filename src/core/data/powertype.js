export const PowerType = {
  ALWAYS: 'always',
  WIND: 'wind',
  SUN: 'sun',
  POWERED: 'powered',
};
