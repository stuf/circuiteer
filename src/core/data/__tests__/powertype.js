import { PowerType } from '../powertype';

test('PowerType', () => {
  expect(PowerType).toMatchSnapshot();
});
