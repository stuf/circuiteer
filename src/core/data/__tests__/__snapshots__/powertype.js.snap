// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PowerType 1`] = `
Object {
  "ALWAYS": "always",
  "POWERED": "powered",
  "SUN": "sun",
  "WIND": "wind",
}
`;
